jenkins/jobs/jobs.yaml

```diff
@@ -175,6 +175,48 @@
       - osci-swift-upload-logs-with-console-plugin:
           upload_source: 'test.log logs'
 
+- job-template:
+    name: 'test-vgpu'
+    node: '{node}'
+
+    wrappers:
+      - timeout:
+          timeout: 30  # Timeout in *minutes*
+          fail: true  # A job run that exceeds the timeout will cause a failure
+      - timestamps
+
+    builders:
+      #- net-info
+      - shell: |
+          #!/bin/bash -x
+          env
+
+          export ZUUL_CHANGES=${ZUUL_CHANGES:-""}
+          export ZUUL_REF=${ZUUL_REF:-""}
+          export ZUUL_CHANGE=${ZUUL_CHANGE:-""}
+          export OPENSTACK_LOGS=/tmp/opentack_logs
+
+          echo "Testing vgpu on xenserver"
+          export VGPU_TEST_LOG_DIR=$WORKSPACE/logs
+          LOG=$WORKSPACE/test.log
+          rm -rf $WORKSPACE
+          mkdir -p $WORKSPACE
+          mkdir -p $VGPU_TEST_LOG_DIR
+          cd $WORKSPACE
+          if [ ! -d qa ]; then
+            git clone https://github.com/citrix-openstack/qa -b master
+          fi
+          cd $HOME/qa/
+          ./test_vgpu.sh $ZUUL_REF $ZUUL_CHANGES 2>&1| tee -a $LOG
+
+          if ! grep -q "VGPU create success" $LOG; then
+              exit 137
```

Comment:
Why 137?

Reply:
No specific meanings, every none zero value is OK. Just use the value which used by other jobs.

Reply:
Fine